### ISSUE TITLE
Trolie Test Server is now available

### ISSUE THREAD

**Issue by travis-qualus** [trolie/spec#120]
I wanted to announce the availability of a TROLIE API test server.

http://dev.trolie.gbcloud.io/

It currently only supports a few basic calls to get people started.  Our hope is to expand it over the next few months to cover the entire spec, and add additional validation checks as those are defined by the various markets.

Feel free to contact me for any help or information - travis.rouillard@qualuscorp.com

**Comment by getorymckeag** [comment]
@travis-qualus thank you for setting this up!  @catkins-miso I suggest that we add a link to this on the website.  Any thoughts?

**Comment by catkins-miso** [comment]
@travis-qualus this is fantastic! I'm working on unblocking the domain here at MISO, but I can access it from my personal machines. I think we should add the familiar "who's using" or similar tout to the trolie.energy homepage @getorymckeag, absolutely, and we should add GEV and MISO's logos. I'll need to ask permission here. Should I reach out to Lance about doing the same?

**Comment by getorymckeag** [comment]
@travis-qualus I think we'll add a link to your server to the website.  Are you cool with that?  I'll send you the PR for comment.

**Comment by travis-qualus** [comment]
Yeah, that would be great.

**Comment by alexthornton** [comment]
> we should add the familiar "who's using" or similar

Any updates on this?

**Comment by travis-qualus** [comment]
At this point, I don't know that anybody has ever used it.  We've never seen any requests or questions about it.  It's hard to justify spending any more time on it if its not needed.

**Comment by alexthornton** [comment]
Thanks for the quick response. Is the test server linked in the documentation? I only learned about it through this issue, so perhaps it's a discoverability issue.

Regarding the "we should add the familiar "who's using" or similar", I interpreted that as more of a "who's using TROLIE" with a logo zoo of companies that have adopted TROLIE. Likely @catkins-miso or @getorymckeag would know best there.

**Comment by travis-qualus** [comment]
Perhaps.  I don't have the ability to post it in the main Trolie page or documentation.

You can reach me directly to discuss - travis.rouillard@qualuscorp.com

**Comment by getorymckeag** [comment]
My bad, I kind of lost the thread on this- also, a "used by" logo set is also a good idea.  I'll create a separate issue for that.
